### PR TITLE
Add an API to get bio page data

### PR DESCRIPTION
**Pull request by shifucun**
This now retrieves portion of the data with node traversal. When cache data is ready, it should use the pre-computed sub-graph from Bigtable.

### CODE REVIEW

File: test/integration/golden_response/get_protein_page_data/p53_human.json [hunk 1]
```diff
@@ -0,0 +1,1711 @@
+{
```
Comment (shifucun) [inline test/integration/golden_response/get_protein_page_data/p53_human.json:1]: @Spaceenter, this is a realistic cache data

File: proto/internal.proto [hunk 2]
```diff
@@ -60,4 +68,25 @@ message GetPlacePageDataResponse {
...
+  message LinkedNodes {
+    string property = 1;
+    PropertyDirection direction = 2;
+    repeated GetProteinPageDataResponse nodes = 3;
+  }
+  // A group of neighbour nodes by predicate.
+  repeated LinkedNodes neighbours = 1;
+  // Value of the node, could be dcid or node string value.
+  string value = 2;
 }
```
Comment (pradh) [inline proto/internal.proto:92]: This seems like a general proto, not protein specific. Can we reuse for others?

Comment (pradh) [reply proto/internal.proto:92]: So, generally, there is a (type, dcid) and response is a graph?

Comment (shifucun) [reply proto/internal.proto]: Moved the proto to common.proto. 

Right, for disease, there will be a separate config and a subgraph tailored for the disease page.

**Comment by shifucun** [comment]
Thanks for review!

**Review by pradh — APPROVED** [review]

File: proto/internal.proto (hunk 2 above)
Comment (shifucun) [reply proto/internal.proto]: Make the API to be more generic with naming changed from protein to bio

**Pull request merged**